src/medium-zoom.js

```diff
@@ -18,24 +18,32 @@ const isNode = selector => selector && selector.nodeType === 1
  *
  * @param {(string|Element[])} selector The selector to target the images to attach the zoom to
  * @param {object} options The options of the zoom
- * @param {number} [options.margin=0] The space outside the zoomed image
- * @param {string} [options.background="#fff"] The color of the overlay
  * @param {number} [options.scrollOffset=48] The number of pixels to scroll to dismiss the zoom
  * @param {boolean} [options.metaClick=true] A boolean to enable the default action on meta click
  * @param {(string|Element|object)} [options.container] The element to render the zoom in or a viewport object
+ * @param {(string|array)} [options.overlayStyles{background="#fff"}] To pass others css styles to overlay with default background-color=#fff
+ * @param {(string|array)} [options.imgStyles] To pass others css styles to image
  * @param {(string|Element)} [options.template] The template element to show on zoom
  * @return The zoom object
  */
 const mediumZoom = (
   selector,
   {
-    margin = 0,
-    background = '#fff',
     scrollOffset = 48,
     metaClick = true,
     container,
+    overlayStyles = {
+      backgroundColor: '#fff'
+    },
+    imgStyles = {},
     template
-  } = {}
+  } = {
+    scrollOffset: 48,
+    metaClick: true,
+    overlayStyles: {
+      backgroundColor: '#fff'
+    }
+  }
```

Comment:
You can move this value to a constant at the beginning of the file, named `DEFAULT_OPTIONS`. Then you can reuse it everywhere.

```js
const mediumZoom = (
  selector,
  {
    scrollOffset = DEFAULT_OPTIONS.scrollOffset,
    metaClick = DEFAULT_OPTIONS.metaClick,
    container = DEFAULT_OPTIONS.container,
    overlayStyles = DEFAULT_OPTIONS.overlayStyles,
    imgStyles = DEFAULT_OPTIONS.imgStyles,
    template = DEFAULT_OPTIONS.template
  } = DEFAULT_OPTIONS
) => {
  ....
```